### ISSUE TITLE
Docker-explorer can't fetch list on MACOSX

### ISSUE THREAD

**Issue by 42Atomys** [formulahendry/vscode-docker-explorer#11]
Hello, 

docker-explorer can't connect to the docker demon on mac.
error: 
`[Failed to list Docker Containers] Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?`
Same if enter `eval $(docker-machine env default)` on docker-explorer terminal

**Comment by formulahendry** [comment]
Run `docker ps` on terminal of you Mac. It that working?

**Comment by AtomGrafiks** [comment]
Yes

**Comment by formulahendry** [comment]
How about running `docker ps` on docker-explorer terminal?

**Comment by AtomGrafiks** [comment]
![screen shot 2017-06-20 at 16 00 44](https://user-images.githubusercontent.com/8448358/27337091-bfc8bb56-55d1-11e7-88c3-5bd58ee9b56b.png)
It's ok on docker-explorer terminal, but not on left UI, if I click on 'reload arrow', extension return `[Failed to list Docker Containers] Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?`

**Comment by formulahendry** [comment]
I have tested the extension on both Windows and Mac. This is weird since code behind is just calling command like `docker ps -a --format "{{.ID}} {{.Names}} {{.Image}} {{.Status}}"` to get the containers info. Another issue #9 is related to Linux which needs root privilege to run docker command, but that seems not needed on Mac.

**Comment by jacobrillema** [comment]
I basically had the same issue with Windows 10 x64, running VS Code 64 bit. I had the extension working earlier when I was running Docker for Windows app out of the box. I decided to go a different route and create my hyperv VM manually with docker-machine. I was able to determine that when Node was executing the docker commands (via command prompt) it didn't have all the docker environment variables available. I was able to duplicate that by just going into command prompt.

The only work around I found was to add the following system wide environment variables in Windows.

DOCKER_TLS_VERIFY=1
DOCKER_HOST=tcp://192.168.99.102:2376
DOCKER_CERT_PATH=C:\Users\myuser\.docker\machine\machines\default DOCKER_MACHINE_NAME=default

This works for me as I have configured my VM to be a static ip address. Now everything seems to be working fine. Not sure how we want to handle this docker-machine setup scenario? I know individual docker commands we can send the host, use tls and the cert path options. We may want to add that into a new settings json file? I will create a new issue for that.
